New flaky test?

```
FAIL: CheckoutListSetPackagingCase.test_list_packaging_ok

Traceback (most recent call last):

  File "/home/travis/build/camptocamp/wms/setup/shopfloor/odoo/addons/shopfloor/tests/test_checkout_change_packaging.py", line 72, in test_list_packaging_ok

    + self.packaging_pallet

  File "/home/travis/build/camptocamp/wms/setup/shopfloor/odoo/addons/shopfloor/tests/common.py", line 258, in assert_response

    "\n\nExpected:\n%s" % (pformat(response), pformat(expected)),

AssertionError: {'data': {'change_packaging': {'picking': {[624 chars]ing'} != {'next_state': 'change_packaging', 'data': [624 chars]}]}}}

  {'data': {'change_packaging': {'package': {'id': 13,

                                             'move_line_count': 1,

                                             'name': 'PACK0000013',

                                             'packaging': {'code': 'DEFAULT',

                                                           'id': 32,

                                                           'name': 'Default Type',

                                                           'qty': 0.0},

                                             'storage_type': None,

                                             'weight': 0.0},

                                 'packaging': [{'code': 'DEFAULT',

-                                               'id': 32,

?                                                      ^

+                                               'id': 34,

?                                                      ^

                                                'name': 'Default Type',

                                                'qty': 0.0},

                                               {'code': 'DEFAULT',

                                                'id': 33,

                                                'name': 'Default Type',

                                                'qty': 0.0},

                                               {'code': 'DEFAULT',

-                                               'id': 34,

?                                                      ^

+                                               'id': 32,

?                                                      ^

                                                'name': 'Default Type',

                                                'qty': 0.0}],

                                 'picking': {'id': 52,

                                             'move_line_count': 1,

                                             'name': 'WH/CHK/00002',

                                             'note': None,

                                             'origin': None,

                                             'partner': {'id': 60,

                                                         'name': 'Customer'},

                                             'scheduled_date': '2020-09-16T14:22:22+00:00',

                                             'weight': 20.0}}},

   'next_state': 'change_packaging'} : 

```

--- a/shopfloor/tests/test_checkout_change_packaging.py
+++ b/shopfloor/tests/test_checkout_change_packaging.py
@@ -5,12 +5,17 @@ class CheckoutListSetPackagingCase(CheckoutCommonCase):
     @classmethod
     def setUpClassBaseData(cls):
         super().setUpClassBaseData()
+        pallet_type = (
+            cls.env["product.packaging.type"]
+            .sudo()
+            .create({"name": "Pallet", "code": "P", "sequence": 3})
+        )
         cls.packaging_pallet = (
             cls.env["product.packaging"]
             .sudo()
             .create(
                 {
-                    "sequence": 3,
+                    "packaging_type_id": pallet_type.id,
                     "name": "Pallet",
                     "barcode": "PPP",
                     "height": 100,
@@ -19,12 +24,17 @@ class CheckoutListSetPackagingCase(CheckoutCommonCase):
                 }
             )
         )
+        box_type = (
+            cls.env["product.packaging.type"]
+            .sudo()
+            .create({"name": "Box", "code": "B", "sequence": 2})
+        )
         cls.packaging_box = (
             cls.env["product.packaging"]
             .sudo()
             .create(
                 {
-                    "sequence": 2,
+                    "packaging_type_id": box_type.id,
                     "name": "Box",
                     "barcode": "BBB",
                     "height": 20,
@@ -33,12 +43,17 @@ class CheckoutListSetPackagingCase(CheckoutCommonCase):
                 }
             )
         )
+        inner_box_type = (
+            cls.env["product.packaging.type"]
+            .sudo()
+            .create({"name": "Inner Box", "code": "I", "sequence": 1})
+        )
         cls.packaging_inner_box = (
             cls.env["product.packaging"]
             .sudo()
             .create(
                 {
-                    "sequence": 1,
+                    "packaging_type_id": inner_box_type.id,
                     "name": "Inner Box",
                     "barcode": "III",
                     "height": 10,

